octobot_trading/personal_data/trades/channel/trades.py

```diff
@@ -36,18 +36,12 @@ async def perform(self, trades, old_trade=False):
                         self.channel.get_filtered_consumers(symbol=symbol):
                     trade_id: str = trade[enums.ExchangeConstantsOrderColumns.ID.value]
 
-                    added: bool = await self.channel.exchange_manager.exchange_personal_data.handle_trade_update(
+                    await self.channel.exchange_manager.exchange_personal_data.handle_trade_update(
                         symbol,
                         trade_id,
                         trade,
-                        should_notify=False)
-
-                    if added:
-                        await self.send(cryptocurrency=self.channel.exchange_manager.exchange.
-                                        get_pair_cryptocurrency(symbol),
-                                        symbol=symbol,
-                                        trade=trade,
-                                        old_trade=old_trade)
```

Comment:
Issue was that we were pushing `trade` to the channel but `trade` is the ccxt parsed trade, not the OctoBot parsed trade.to_dict()